src/TesApi.Web/BatchScheduler.cs

```diff
@@ -1197,15 +1208,14 @@ static IEnumerable<string> MungeCleanupScriptForContainerConfig(IEnumerable<stri
         /// Uploads the file if content is provided.
         /// </summary>
         /// <param name="inputFile"><see cref="TesInput"/> file</param>
-        /// <param name="taskId">TES task Id</param>
+        /// <param name="task">TES task</param>
         /// <param name="queryStringsToRemoveFromLocalFilePaths">Query strings to remove from local file paths</param>
         /// <returns>List of modified <see cref="TesInput"/> files</returns>
-        private async Task<TesInput> GetTesInputFileUrl(TesInput inputFile, string taskId, List<string> queryStringsToRemoveFromLocalFilePaths)
+        private async Task<TesInput> GetTesInputFileUrl(TesInput inputFile, TesTask task, List<string> queryStringsToRemoveFromLocalFilePaths)
         {
-            if (inputFile.Path is not null
-                && !inputFile.Path.StartsWith($"{CromwellPathPrefix}/", StringComparison.OrdinalIgnoreCase))
+            if (string.IsNullOrWhiteSpace(inputFile.Path) || !inputFile.Path.StartsWith("/"))
             {
-                throw new TesException("InvalidInputFilePath", $"Unsupported input path '{inputFile.Path}' for task Id {taskId}. Must start with '{CromwellPathPrefix}'.");
+                throw new TesException("InvalidInputFilePath", $"Unsupported input path '{inputFile.Path}' for task Id {task.Id}. Must start with '/'.");
```

Comment:
I'm sure you saw, there's a test that checks this string -_-

Reply:
I didn't see one in this method